packages/api-headless-cms/src/plugins/graphql/installResolver/install.ts

```diff
@@ -0,0 +1,92 @@
+import mdbid from "mdbid";
+import { ErrorResponse, Response } from "@webiny/graphql";
+import { GraphQLFieldResolver } from "@webiny/graphql/types";
+
+const initialEnvironment = {
+    id: mdbid(),
+    name: "Production",
+    description: "Ready to go live",
+    createdFrom: null
+};
+
+const initialEnvironmentAlias = {
+    slug: "production",
+    name: "Production",
+    description: "Ready to go live"
+};
+
+const initialContentModelGroup = {
```

Comment:
We don't want to have a generic group for "e-commerce". The "e-commerce" is completely irrelevant to the headless CMS app.

Let's go with something like "Ungrouped". 

@Pavel910 @SvenAlHamad If you have any other suggestions, feel free to jump in.